openhtf/util/functions.py

```diff
@@ -18,28 +18,37 @@
 import inspect
 
 
-def CallOnce(func):
+def CallOnce(func=None, allow_args=False):
```

Comment:
I really don't like this fake memoization, even if it's explicitly invoked via the `allow_args` keyword.  At one point I thought we had `LogOnce` functionality, but I can't seem to find it.  How about implementing that instead of a generic utility function like this?